pycqed/measurement/multi_qubit_module.py

```diff
@@ -155,6 +155,29 @@ def get_operation_dict(qubits):
     return operation_dict
 
 
+def get_multiplexed_readout_detector_function(qubits, det_type, **kw):
```

Comment:
I renamed it now to get_n_qubit_detector_function. This is a wrapper around get_multiplexed_readout_detector_functions. I would keep it distinct from the latter since I wouldn't want the default values for nr_averages and nr_shots to be taken from the qubits. But maybe it makes sense? I'm not sure. What do you think, should we combine the two functions?